packages/@okta/vuepress-site/conductor.yml

```diff
@@ -1463,3 +1463,5 @@ redirects:
     to: /docs/guides/sign-users-out/springboot/sign-out-of-okta/
   - from: /guides/sign-users-out/springboot/sign-out-of-your-app/index.html
     to: /docs/guides/sign-users-out/springboot/sign-out-of-your-app/
+  - from: /use_cases/promote-oan-integration/index.html
+  - to: https://www.okta.com/integrate/documentation/promotion/
```

Comment:
The `to` line shouldn't have the `-`.  It's part of the same entry as the `from` line